circle.yml

```diff
@@ -13,4 +13,4 @@ deployment:
       - git config --global user.name "Apiary Bot"
       - git config --global user.email "support@apiary.io"
       - git remote set-url origin "https://${GH_TOKEN}@github.com/apiaryio/apiaryio.github.com"
-      - rake deploy
+      - bundle exec rake deploy
```

Comment:
Is this needed? When I wrote the deploy phase originally, I designed it to run itself inside Bundler so the user didn't need to run it inside bundler.

For example, we use bundle exec the deploy phase: https://github.com/apiaryio/apiaryio.github.com/blob/master/Rakefile#L88